src/core/Program.js

```diff
@@ -105,6 +105,8 @@ export class Program {
         for (let aIndex = 0; aIndex < numAttribs; aIndex++) {
             const attribute = gl.getActiveAttrib(this.program, aIndex);
             const location = gl.getAttribLocation(this.program, attribute.name);
+            if (location === -1) continue;
```

Comment:
Do you have a use case when location would be `-1` ? This is an iteration of active attributes, so in my understanding this should never occur.

Reply:
Here is example: https://codepen.io/RAX7/pen/ExEazeW

Reply:
ah I see, thanks for that, that makes sense